packages/expo-gl/src/Canvas.tsx

```diff
@@ -1,14 +1,15 @@
 import * as React from 'react';
 import { findDOMNode } from 'react-dom';
 import {
-  createElement,
   LayoutChangeEvent,
   PixelRatio,
   StyleSheet,
   View,
   ViewProps,
 } from 'react-native';
 
+const createElement = require('react-native').unstable_createElement || require('react-native').createElement;
```

Comment:
We disallow `require(`react-native`)` because this breaks web tree-shaking, effectively bundling all of `react-native`.